app/controllers/trucks_controller.rb

```diff
@@ -4,4 +4,10 @@ class TrucksController < ApplicationController
   def index
     @trucks = Truck.all
   end
+
+  # GET /trucks/1
```

Comment:
I am pretty meh about explanatory comments such as this one. Good Ruby code is largely self-documenting, and how much value does this really add?